unyt/tests/test_array_functions.py

```diff
@@ -199,3 +199,83 @@ def test_histogramdd():
     assert 1 * xbins.units == 1 * x.units
     assert 1 * ybins.units == 1 * y.units
     assert 1 * zbins.units == 1 * z.units
+
+
+def test_concatenate():
+    x1 = np.random.normal(size=100) * cm
+    x2 = np.random.normal(size=100) * cm
+    res = np.concatenate((x1, x2))
+    assert 1 * res.units == 1 * cm
+    assert res.shape == (200,)
+
+
+def test_concatenate_different_units():
+    x1 = np.random.normal(size=100) * cm
+    x2 = np.random.normal(size=100) * s
+    with pytest.raises(RuntimeError, match=r"Your arrays must have identical units\."):
```

Comment:
Forgot to update the error here. FWIW I strongly encourage you to run the tests locally before you push to github, it's pretty fast.

Reply:
https://unyt.readthedocs.io/en/stable/contributing.html#testing-unyt

Reply:
I thought I did actually... anyway it's even more broken than just this (see above thread)